src/Workspaces/Remote/Core/EditAndContinue/ManagedHotReloadLanguageService.cs

```diff
@@ -304,9 +304,10 @@ public async ValueTask<ManagedHotReloadUpdates> GetUpdatesAsync(ImmutableArray<s
                     Location.None,
                     string.Format(descriptor.MessageFormat.ToString(), "", e.Message));
 
+                var firstProject = designTimeSolution.GetProject(runningProjectIds.FirstOrDefault()) ?? designTimeSolution.Projects.First();
```

Comment:
same idea.  if runningProjIds is always expected to be non-empty, i can def simplify this.